src/main/java/com/twilio/oai/TwilioJavaGenerator.java

```diff
@@ -144,10 +144,26 @@ public OperationsMap postProcessOperationsWithModels(final OperationsMap objs, L
         final OperationsMap results = super.postProcessOperationsWithModels(objs, allModels);
         final List<CodegenOperation> opList = directoryStructureService.processOperations(results);
         JavaApiResources apiResources = processCodegenOperations(opList);
+        apiResources.authMethodPackage = processAuthMethods(opList);
         results.put("resources", apiResources);
         return results;
     }
 
+    private String processAuthMethods(List<CodegenOperation> opList) {
+        String authMethodPackage = "";
+        if(opList != null){
+            List<CodegenSecurity> authMethods = opList.get(0).authMethods;
+            if(authMethods != null){
+                for(CodegenSecurity c : authMethods){
+                    if(c.isOAuth == true){
+                        authMethodPackage = ".bearertoken";
+                    }
+                }
+            }
+        }
+        return authMethodPackage;
```

Comment:
We have to process this for 3 types of auth.
3 types of auth: basic, bearer token, no auth.
Add **processAuthMethods** in JavaApiResourceBuilder as we are building ApiResource.

Reply:
Made these changes